### DISCUSSION TITLE
"Contact us" on every page of the documentation

### DISCUSSION THREAD

**Discussion by civier** [neurodesk/neurodesk.github.io#130] [Ideas]
Many people will land in a specific page of NeuroDesk documentation (e.g., if they are provided a link by their sys admin), and won't pass through the front page (https://neurodesk.github.io/). In this situation, they might not be aware that the link to the "Discussions" is only on the front page.
I suggest to put a "contact us" link on the top or left bar (or any other part of the documentation pages that is visible) that leads to the Github discussions forum. And I think that the link should still mention that "(Github account required)".

**Comment by stebo85** [comment]
Dear @civier 

 I added this just now.
